Formula/libvisual-plugins-0.4.rb

```diff
@@ -0,0 +1,40 @@
+class LibvisualPlugins04 < Formula
+  desc "Essential Libvisual 0.4.x plug-ins"
+  homepage "https://github.com/Libvisual/libvisual"
+  url "https://github.com/Libvisual/libvisual/releases/download/libvisual-plugins-0.4.2/libvisual-plugins-0.4.2.tar.gz"
+  sha256 "55988403682b180d0de5e6082f804f3cf066d9a08e887b10eb6a315eb40d9f87"
+  license all_of: ["GPL-2.0-or-later", "LGPL-2.1-or-later"]
+  # head "https://github.com/Libvisual/libvisual", branch: "0.4.x"
+
+  depends_on "bison" => :build
+  depends_on "pkg-config" => :build
+  depends_on "jack"
+  depends_on "libvisual-0.4"
+  depends_on "portaudio"
+
+  on_linux do
+    depends_on "alsa-lib"
+    depends_on "libx11"
+    depends_on "libxext"
+    depends_on "libxrender"
+    depends_on "mesa"
+    depends_on "mesa-glu"
+    depends_on "pulseaudio"
+  end
+
+  def install
+    args = [
+      "--prefix=#{prefix}",
+      "--disable-gdkpixbuf-plugin",
+      "--disable-gforce",
+      "--disable-gstreamer-plugin",
+    ]
+    system "./configure", *args
+    system "make", "-j2"
+    system "make", "install"
+  end
+
+  test do
+    system "sh", "-c", "lv-tool-0.4 --plugin-help 2>&1 | grep -q portaudio"
```

Comment:
Use `assert_match` instead of `grep`. Though, we should use a test that does more than `--help`.